nix/overlay.nix

```diff
@@ -0,0 +1,14 @@
+let
+  srcDef = builtins.fromJSON (builtins.readFile ./nixpkgs-stable.json);
```

Comment:
I think the idea behind this was to also check current lorri against nixpkgs 19.09, so shouldn’t there be a second overlay for testing current master of this repository, building it against 19.09? I might be missing the point.

Reply:
Oops. You are indeed correct.

Do you want two overlays-tests, or just the one (for 19.09)?

Reply:
@Profpatsch ^